charters/Architecture/charter.md

```diff
@@ -0,0 +1,35 @@
+# DCP Architecture Team
+
+## Description
+
+The Data Coordination Platform (DCP) Architecture Team is responsible for the technical oversight of the DCP, including identifying and addressing *cross-cutting* architectural concerns across DCP projects.
+
+The core team is composed of *Technical Leads* representing each DCP component to ensure a diversity of skills and views needed for community technical decision-making. 
```

Comment:
Need technical leads from non-component (ie, non-implementation) teams that could intersect with the "technology oversight" role, eg,
* devsecops team lead
* system integration / release / CI/CD team lead
* schema/metadata team team
etc.